README.md

```diff
@@ -1,70 +1,98 @@
-# Getting Started with Create React App
+# Math-Magician 
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-## Available Scripts
+# Project Description
 
-In the project directory, you can run:
+Delete all the boilerplate from CRA (text, images, styles).
+Create a directory called components.
+Inside components, create a new Calculator.js file.
+In Calculator.js, create a React component that matches this design:
 
-### `npm start`
+# Project Video link:
+[link to video]
```

Comment:
- [ ] [OPTIONAL] Well done @Enekwestanley 👍🏽 Your README is well documented, however, kindly remove this link and add it back when a video link is available.